website/docs/docs/build/python-models.md

```diff
@@ -16,26 +16,20 @@ We encourage you to:
 - Weigh in on [next steps for Python models, beyond v1.3](https://github.com/dbt-labs/dbt-core/discussions/5742).
```

Comment:
sugg: remove lines 5 - 9 above

reason: unnecessary since docusaurus automatically builds a right sidebar (page toc) for all H2s and H3s